src/components/MerchantsPage/gam/CampaignListItem.tsx

```diff
@@ -92,13 +94,16 @@ const CampaignListItem = (props: Props) => {
           <Description>
             {campaign.description}{' '}
             {distributor && (
-              <a
-                href={distributor.website_url}
-                target="_blank"
-                rel="noopener noreferrer"
-              >
-                Learn more about {distributor.name}.
-              </a>
+              <span>
+                Learn more about{' '}
+                <a
+                  href={distributor.website_url}
```

Comment:
This can come from the pair as well, but no need to change it now

Reply:
Rather change the distributor stuff all in one go than be inconsistent

Reply:
do one of you want to take this ticket? it encompasses all this work:
https://trello.com/c/wMetd5qi/708-megagam-migrate-campaign-seller-distributor-fields-to-use-the-campaignssellersdistributors-join-table